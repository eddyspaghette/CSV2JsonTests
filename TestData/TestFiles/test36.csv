ッ!あ!が!ざ!℁!⏳
ツ!ぃ!ぎ!せ!℟!▓
ヅ!い!け!₠!Å!○
ミ!ぅ!げ!₡!ℬ!☝
デ!う!こ!₢!!⭐
ト!ぇ!ご!₣!ψ!😵
ド!え!さ!℀!ↂ!🍕
!ぃ!ぎ!せ!℟!▓
デ!う!こ!₢!Δ!⭐
ト!ぇ!ご!₣!ψ!😵
