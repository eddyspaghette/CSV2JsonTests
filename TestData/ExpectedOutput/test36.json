[
 {
   "ッ!あ!が!ざ!℁!⏳": "ツ!ぃ!ぎ!せ!℟!▓"
 },
 {
   "ッ!あ!が!ざ!℁!⏳": "ヅ!い!け!₠!Å!○"
 },
 {
   "ッ!あ!が!ざ!℁!⏳": "ミ!ぅ!げ!₡!ℬ!☝"
 },
 {
   "ッ!あ!が!ざ!℁!⏳": "デ!う!こ!₢!!⭐"
 },
 {
   "ッ!あ!が!ざ!℁!⏳": "ト!ぇ!ご!₣!ψ!😵"
 },
 {
   "ッ!あ!が!ざ!℁!⏳": "ド!え!さ!℀!ↂ!🍕"
 },
 {
   "ッ!あ!が!ざ!℁!⏳": "!ぃ!ぎ!せ!℟!▓"
 },
 {
   "ッ!あ!が!ざ!℁!⏳": "デ!う!こ!₢!Δ!⭐"
 },
 {
   "ッ!あ!が!ざ!℁!⏳": "ト!ぇ!ご!₣!ψ!😵"
 }
]